subgraphs/aave-v2-forks/schema.graphql

```diff
@@ -545,6 +545,9 @@ type Market @entity {
 
   " Aditional tokens that are given as reward for position in a protocol, usually in liquidity mining programs. e.g. SUSHI in the Onsen program, MATIC for Aave Polygon "
   rewardTokens: [RewardToken!]
+
+  " Aux: Timestamp of the last time rewards were calculated for this market. "
+  _lastRewardsUpdated: BigInt
```

Comment:
this is the standard I have been using in this directory: https://github.com/messari/subgraphs/blob/master/subgraphs/aave-v2-forks/schema.graphql#L317

Reply:
Oh, sounds good. Changed!

Reply:
I also meant to remove the underscore, what do you think?

Reply:
Hmm 🤔 
I added the underscore as that's how I've seen it done in other subgraphs. I think the underscore is convenient since you might not always be looking at where exactly the variable is defined in the schema. For example when you are querying the subgraph from the graphql web client, or trusting some autocomplete in your IDE.

I'll remove the underscore for now because having mixed names (some with and some without underscore) on the same subgraph might be even more confusing 😅  But I think we should use underscores or some other identifier in general, it doens't hurt and adds clarity.

Reply:
I agree, not sure why I did that when building the aave forks. I will make a note for me to fix that after this is merged